docs/tutorials/01_algorithms_introduction.ipynb

```diff
@@ -298,20 +302,13 @@
    "source": [
     "import qiskit.tools.jupyter\n",
     "%qiskit_version_table\n",
```

Comment:
I imagine this version table will pick up the new one when these are run in CI - they could be re-run locally to check-in something that was a bit more current and I think should list qiskit-algorithms in the versions there too. I guess we can see in the CI docs artifact when it all built.

Reply:
It does

![image](https://github.com/qiskit-community/qiskit-algorithms/assets/40241007/7eaa0b92-4003-4c77-b058-830ab9c1a722)

Reply:
yes, this is part of the issue why I have kept the PR a draft. My jupyter environment didn't pick up the right versions (this is why some notebooks failed too). I'll look into it now.